tests/sentry/integrations/jira/test_integration.py

```diff
@@ -406,7 +406,10 @@ def get_client():
                             'string_max_length=self.string_max_length)\n\nmessage\n{code}'
                             ) % (
                                 group.qualified_short_id,
-                                absolute_uri(group.get_absolute_url()),
+                                absolute_uri(
+                                    group.get_absolute_url(
+                                        params={
+                                            'referrer': 'jira_integration'})),
```

Comment:
Urgh, this got so ugly, is this what autopep8 does?

Reply:
yeah it's auto. i think it's nice :P